src/main/kotlin/org/b12x/gfe/core/model/parsers/ParserHlaData.kt

```diff
@@ -0,0 +1,26 @@
+package org.b12x.gfe.core.model.parsers
+
+import com.beust.klaxon.JsonArray
+import com.beust.klaxon.Parser
+import okhttp3.Response
+import org.b12x.gfe.core.model.DataDownload
+
+object ParserHlaData {
```

Comment:
Should this be called `HlaDataParser`?

Reply:
I also have ParserVersionData class, and there will be a ParserKirData class. I tend to group things by type then add the specific details. This could be overkill though, they're in a package called `parsers`. I prefer it because it helps me keep track of what I'm looking at when I have a dozen files open in the IDE. If you would prefer, I can remove the word parser and just have HlaData, VersionData, and KirData, or even just Hla, Version, and Kir. Please let me know if you have a preference.